src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <!-- Keep these in sync with _ILLinkTasksTFM in Sdk.props. -->
     <!-- Keep the netcoreapp TFM in sync with the Mono.Linker.csproj condition below. -->
-    <TargetFrameworks>netcoreapp3.0;net472</TargetFrameworks>
+    <TargetFrameworks>net5.0;net472</TargetFrameworks>
```

Comment:
I recommend to use an msbuild property for the used .NETCoreApp version to keep changes as small as possible in the future.

Reply:
Good idea - we should probably do that as a follow-up - the plan is to try to backport this to 5.0 release, so I would like to keep the changes to minimum.